### PR TITLE
Update traits

### PR DESCRIPTION
**Pull request by ybw0014**
I apologize for my bad git management. Open this new PR for future updates.

This PR is to update some misinformation in traits' descriptions and rename some methods.

### CODE REVIEW

**Comment by Sefiraat** [comment]
> I apologize for my bad git management. Open this new PR for future updates.

You’ve seen my code and my commits/git management - you have nothing to apologise for :D I have no idea what I’m doing at the best of times lol. I’ll keep an eye on this ha :)

**Comment by ybw0014** [comment]
Does silver sing plate actually heal or just cancel lightning damage? I don't find any heal code in `plateSingSilver`.

**Comment by Sefiraat** [comment]
Ill

> Does silver sing plate actually heal or just cancel lightning damage? I don't find any heal code in `plateSingSilver`.

I’ll check my notes when I’m home but I’m fairly sure it’s meant to heal. But I’ll do that one as I want to make a generic EntityUtil for healing LivingEntities and replace all my heals with it, I’ll include this at the same time!

**Comment by ybw0014** [comment]
> Ill
> 
> > Does silver sing plate actually heal or just cancel lightning damage? I don't find any heal code in `plateSingSilver`.
> 
> I’ll check my notes when I’m home but I’m fairly sure it’s meant to heal. But I’ll do that one as I want to make a generic EntityUtil for healing LivingEntities and replace all my heals with it, I’ll include this at the same time!

Also, the tin sing will heal when poisoned.

**Comment by ybw0014** [comment]
The zinc sing rod will give levitation 2 while sneaking.

**Comment by Sefiraat** [comment]
Sorry, i'm so used to deleting my branches after a merge. I merged my changes in early due to a possible protection dupe reported :)

**Comment by ybw0014** [comment]
So for the brightburn trait, should this trait repel all hostile mobs?

**Comment by Sefiraat** [comment]
> So for the brightburn trait, should this trait repel all hostile mobs?

Yes, should be monster.. sigh.

**Comment by ybw0014** [comment]
For the gleam trait (metal sing links), does it really reflect damage back?

**Comment by Sefiraat** [comment]
> For the gleam trait (metal sing links), does it really reflect damage back?

It should, but it doesn't. Sigh.. I swear I may delete Intellj ;P

I have just pushed to Dev - thanks as ever xD

**Comment by ybw0014** [comment]
for the Cleave trait:
1. Does it damage all nearby entities **including the player**? 
2. Is the cleave damage halved? (In-game yes, in wiki no, in code no)

**Comment by ybw0014** [comment]
for the carbon mesh binder, does it reduced damage taken by 33%?

for the carbon mesh gambeson, is the damage modifier missing?

**Comment by ybw0014** [comment]
for the Soul Siphon trait, does it really have a cap?

File: src/main/java/io/github/sefiraat/slimetinker/events/EntityDamageEvents.java [hunk 1]
```diff
@@ -383,14 +389,14 @@ public static void plateMagic(EventFriend friend) {
...
-        if (friend.getDamagedEntity() instanceof LivingEntity) {
+        if (friend.getDamagedEntity() instanceof Monster) {
             LivingEntity e = (LivingEntity) friend.getDamagedEntity();
             ItemStack i = friend.getActiveStack();
             ItemMeta im = i.getItemMeta();
             NamespacedKey k = SlimeTinker.inst().getKeys().getArmourSoulsStored();
             Validate.notNull(im, "Meta is not null, this is odd!");
             long souls = PersistentDataAPI.getLong(im, k ,0);
-            friend.setDamageMod(friend.getDamageMod() + ((double) souls / 100L));
+            friend.setDamageMod(friend.getDamageMod() + ((double) souls / 10000L));
```
Comment (ybw0014) [inline src/main/java/io/github/sefiraat/slimetinker/events/EntityDamageEvents.java:399]: It should be 1% dmg per 100 souls, so divides by 10k

File: src/main/java/io/github/sefiraat/slimetinker/events/EntityDamageEvents.java [hunk 2]
```diff
@@ -383,14 +389,14 @@ public static void plateMagic(EventFriend friend) {
     }
 
     public static void linksSegganesson(EventFriend friend) {
-        if (friend.getDamagedEntity() instanceof LivingEntity) {
+        if (friend.getDamagedEntity() instanceof Monster) {
```
Comment (ybw0014) [inline src/main/java/io/github/sefiraat/slimetinker/events/EntityDamageEvents.java:392]: For hostile mobs it should me `Moster`

**Review by Sefiraat — APPROVED** [review]

**Comment by ybw0014** [comment]
still need to change some description in traits.

**Pull request merged**